docker-compose.yml

```diff
@@ -8,10 +8,14 @@ services:
       - EQ_AUTHOR_API_URL=http://docker.for.mac.host.internal:4000/graphql
     ports:
       - "9000:9000"
+      - 5858:5858 # open port for debugging
```

Comment:
add `"` to keep it consistent

Reply:
my bad... copied from author-api where we don't use quotes. fixed